lib/gitlab_post_receive.rb

```diff
@@ -1,5 +1,6 @@
 require_relative 'gitlab_init'
```

Comment:
@david-martin Should the changes in this file be committed, or was it just debug logging that you added for development purposes when trying to get this to work?